features/buyer/buyer_clarification.feature

```diff
@@ -1,4 +1,4 @@
-@buyer @buyer-clarification
+@buyer @buyer-clarification @skip-staging @skip-production
```

Comment:
Out of curiosity, why is this skipping these stages now and not before?

Reply:
We need to skip them now because staging and production will be different to preview. If we make the test pass on preview, it will fail on staging.

Reply:
We'll take the skip tags out once the change has been deployed.

Reply:
These tests shouldn't get run on production so we probably don't need a skip tag for it (otherwise everything in this module would end up with a permanent `@skip-production`).

Reply:
@katstevens OK I made this change now in 8b2a313